utils/timed_version.go

```diff
@@ -0,0 +1,142 @@
+package utils
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+	_ "unsafe"
```

Comment:
Is this used?

Reply:
it's required for linkname to get the monotonic time function from runtime

Reply:
added a comment to clarify